packages/explorer/src/components/playground/CreateNew.tsx

```diff
@@ -1,10 +1,10 @@
-import { Dispatch, SetStateAction, useEffect, useState } from 'react'
-import { IoMdRemoveCircleOutline } from 'react-icons/io'
-import { Computer } from '@bitcoin-computer/lib'
-import { UtilsContext } from '@bitcoin-computer/components'
-import { TypeSelectionDropdown } from '../TypeSelectionDropdown'
-import { getErrorMessage, getValueForType, isValidRev, sleep } from '../../utils'
-import { ModSpec } from './Modspec'
+import { Dispatch, SetStateAction, useEffect, useState } from "react"
```

Comment:
It looks like in this file, in ExecuteExpression.tsx, and in Playground.tsx, the lint-fix is changing from single colon to semicolon. We should unify the style (in all packages) and use single colon.